Makefile

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env make -f
+# This Makefile assumes an activated Python virtual environment.
```

Comment:
So, the assumption is that the GitHub Actions workflow is in charge of setting up the virtual environment and installing the dependencies, right? And a local build may or may not be producing the same artifact?

Reply:
> So, the assumption is that the GitHub Actions workflow is in charge of setting up the virtual environment and installing the dependencies, right?

Well, yes:

https://github.com/jenstroeger/python-package-template/blob/7292ee9804ba6cae77b63726ae9691e9aeb154e4/.github/workflows/pull-request.yml#L34-L43

> And a local build may or may not be producing the same artifact?

Unfortunately, yes, I think. The user of the local venv needs to make sure that the venvs and the venv setup are in sync: after modifying `setup.py` to update dependent package versions I need to [`pip install --upgrade`](https://pip.pypa.io/en/stable/user_guide/#only-if-needed-recursive-upgrade) and  [`tox --recreate`](https://tox.wiki/en/latest/config.html#cmdoption-tox-r) to pick up the new package.